### PR TITLE
practice merging a branch via pull request

### PR DESCRIPTION
**Pull request by jveregge**
This resolves something or other.

### CODE REVIEW

**Comment by jveregge** [comment]
@IbraheemYSaleh oh please look at my stuff...

**Comment by IbraheemYSaleh** [comment]
Can't go wrong with liking cats! Commit this bio immediately!

File: jveregge-bio.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Bald, old, cranky and likes cats. :octocat:
```
Comment (IbraheemYSaleh) [inline jveregge-bio.md]: My kind of guy.

Comment (IbraheemYSaleh) [inline jveregge-bio.md]: Is this different than the other comment?

**Pull request merged**